shuup/default_reports/reports/total_sales.py

```diff
@@ -23,15 +23,27 @@ class TotalSales(OrderReportMixin, ShuupReportBase):
         {"key": "name", "title": _("Shop Name")},
         {"key": "currency", "title": _("Currency")},
         {"key": "order_amount", "title": _("Order Amount")},
+        {"key": "customers", "title": _("Customers")},
+        {"key": "customer_avg_sale", "title": _("Customer Average Sale")},
         {"key": "total_sales", "title": _("Total Sales")},
     ]
 
     def get_data(self):
         orders = self.get_objects()
-        total_sales_value = orders.aggregate(total=Sum("taxful_total_price_value"))["total"] or 0
+        aggregation = orders.aggregate(customers=Count("customer", distinct=True),
+                                       total=Sum("taxful_total_price_value"))
+        customer_avg_sale = orders.values("customer").annotate(
+            average=Avg("taxful_total_price_value")
+        ).order_by().aggregate(avg=Avg("average"))["avg"] or 0
```

Comment:
Are you sure we want to take the average of averages? Shouldn't we *just* be aggregating here?